scripts/merge-vs-deps.ps1

```diff
@@ -1,11 +1,9 @@
 [CmdletBinding(PositionalBinding=$false)]
-Param(
-    [string]$userName,
-    [string]$userEmail,
-    [string]$accessToken = "")
+Param([string]$accessToken = "")
 
-git config user.name $userName
-git config user.email $userEmail   
+# name and email are only used for merge commit, it doesn't really matter what we put in there.    
+git config user.name "RoslynValidation"
+git config user.email "validation@roslyn.net"   
```

Comment:
Let's use something ending in `@microsoft.com` or before we discover that this does matter and the poor owner of roslyn.net is surprised. :smile:

Reply:
(We do own roslyn.io if you want to be hip and use that.)

Reply:
@dotnet-bot has dotnet-bot@dotnetfoundation.org :)